src/screens/sync-account/after-sync-qr-scan/confirm-sync/confirm-sync.tsx

```diff
@@ -50,7 +54,12 @@ export const ConfirmSync: FC<ConfirmSyncProps> = ({ onSubmit }) => {
           <View>
             <Divider size={formatSize(12)} />
             <Label label="Password" description="The same password is used to unlock your extension." />
-            <FormPasswordInput name="password" />
+            <FormPasswordInput
```

Comment:
Check please condition, error message is showing up even password correct after close and open app